test_scripts/WebEngine/WebEngineProjection/001_Registration.lua

```diff
@@ -33,10 +34,12 @@ config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
 --[[ Scenario ]]
 common.Title("Preconditions")
 common.Step("Clean environment", common.preconditions)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
+    { appSessionId, appHMIType })
```

Comment:
```suggestion
  { appSessionId, appHMIType })
```
Please remove redundant indents

Reply:
@VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/e1b8a7c00153273c0320dc9b9b19fe8f2c9a6410#diff-4f3559029bee4e49c37fb96dd1163cc6R38)